GAE/src/com/gallatinsystems/framework/dao/BaseDAO.java

```diff
@@ -97,9 +101,12 @@ public void setDomainClass(Class<T> e) {
      */
     public <E extends BaseDomain> E save(E obj) {
         PersistenceManager pm = PersistenceFilter.getManager();
+        Long who = org.waterforpeople.mapping.app.web.CurrentUserServlet.getCurrentUserId();
         obj.setLastUpdateDateTime(new Date());
+        obj.setLastUpdateUserId(who);
```

Comment:
What would happen if user is null?

Reply:
Then we would store that null, which is what we have been storing since forever anyway.